document/js-api/index.bs

```diff
@@ -729,9 +729,20 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
 
 <h3 id="globals">Globals</h3>
 
+<pre class="idl">
+enum ValueType {
+  "i32",
+  "i64",
+  "f32",
+  "f64"
+};
+</pre>
+
+Note: this type may be extended with additional cases in future versions of WebAssembly. Client code should not assume that no other constants can occur.
```

Comment:
"should" is an RFC2119 term, so must not be used in non-normative text, including notes.

Reply:
Good catch, can I rephrase it to "We advise client code not to assume...."?

Reply:
Let's just delete this sentence.

Reply:
Before I do the change, cc @rossberg do we agree to delete the last sentence?

Reply:
If you feel that the warning is sufficiently clear without it and can guarantee that the enum will not become a "breaking change" trap for us later then I'm fine.